.github/workflows/atlantis-base.yml

```diff
@@ -0,0 +1,34 @@
+name: atlantis-base
+
+on:
+  push:
+    paths:
+      - 'docker-base/**'
+    branches:
+      - "master"
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: docker-base
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Login to Packages Container registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - run: echo "TODAY=$(date +"%m%d%Y")" >> $GITHUB_ENV
```

Comment:
should we use `date +"%b-%d-%Y" | tr '[A-Z]' '[a-z]'` so it looks like `jun-22-2021`?

Reply:
I am trying to the [calver](https://calver.org/) in here, in that sense, it should use numeric value, maybe I can do `2021.06.22` rather than `06222021` or `jun-22-2021`, what do you think?